packages/react-atlas-core/src/avatar/Avatar.js

```diff
@@ -6,32 +6,62 @@ import themeable from 'react-themeable';
  *
  * **NOTE**: children will always take precedence over props passed into component.
  */
-const Avatar = ({children, icon, image, title, ...props}) => {
-  const theme = themeable(props.theme);
-  let kids = children;
-  if (React.Children.count(children) === 1 && typeof children === "string"){
-    kids = <span {...theme(4, 'letter')}>{children[0]}</span>;
-  }
+ class Avatar extends React.Component {
+   constructor(props) {
+     super(props);
 
-  let avatar = null;
+     let image;
 
-  if (typeof image === "string") {
-    avatar = <img src={image} title={title} {...theme(3, 'image')} />;
-  } else if (image){
-    avatar = image;
-  } else if (icon){
-    avatar = icon;
-  } else if (title) {
-    avatar = <span {...theme(2, 'letter')}>{title[0]}</span>;
-  }
+     if(typeof props.image === undefined) {
+       if(typeof props.defaultImage === undefined) {
+         image = null;
+       } else {
+         image = props.defaultImage;
+       }
+     } else {
+       image = props.image;
+     }
 
-  return (
-    <div {...props} {...theme(1, 'avatar')}>
-      {kids}
-      {avatar}
-    </div>
-  );
-};
+     this.state = {image: image};
```

Comment:
Not necessary, but just FYI you could use ES6 Object Literal Shorthand and just do `{image}`